app/views/provider_interface/reports/withdrawal_reports/show.html.erb

```diff
@@ -13,6 +13,9 @@
       <p class="govuk-body">
         You’ll be able to see this report once it contains data from at least <%= ProviderReports::MINIMUM_DATA_SIZE_REQUIRED %> candidates. This is to protect the privacy of candidates.
       </p>
+      <p class="govuk-body">
+        This data has only been collected from 11 April 2023.
```

Comment:
Did we also want to clarify that we're only showing values for _structured_ withdrawal reasons?

Reply:
I'm not sure users would know what that means? We could say:

The report shows data from candidates who withdrew their application and selected their reason from a set list. This is an optional question. Data for this report has only been collected from 11 April 2023.

Reply:
i like.

Reply:
only took me about 2 months to come up with that.

Reply:
can't put a price on perfection eh